optuna/integration/skorch.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any
+
+import optuna
+
+with optuna._imports.try_import() as _imports:
+    from skorch.callbacks import Callback
+    from skorch.net import NeuralNet
+
+if not _imports.is_successful():
+    Callback = object  # NOQA
+    NeuralNet = object  # NOQA
+
+
+class SkorchPruningCallback(Callback):
```

Comment:
How do we make this class "experimental" using the `experimental` decorator?

Reply:
What do you mean by that?

Reply:
Sorry for my grammatical mistake. I mean: how about using the experimental decorator for this newly introduced class?
```suggestion
@experimental("2.1.0")
class SkorchPruningCallback(Callback):
```

Reply:
As per our internal discussion which is now published in https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#experimental-feature, I don't want to say this feature as "experimental".

Reply:
I got it. So this class is stable and there is no room for the interface to change at all? I think it's OK that we do not mark this class as "experimental".

By the way, how about other integration pruning callback classes? For example, I wonder whether the experimental decorator for the `CatalystCallback` can be removed.